tests/jsTests/menuSpec.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Suite : Tests for showing and hiding of menu
+ * Description : This is a test suite for showing and hiding menu for smaller screens.
```

Comment:
If you are writing comments, try to give information which already not there. Think of yourself as someone who is seeing the code for the first time, what info would help him understand the code. Whatever info is already there, needs not be repeated. There could possibly be something more, to this test. If you really don't have anything more to add, then please don't write the comments. One possible thing could be  answering this ques: why only for smaller screens? Think more on it.